autoload/codeclimate.vim

```diff
@@ -92,25 +92,30 @@ function! s:RunAnalysis(files)
     echohl None
     return
   endif
+  let l:issues = s:ParseIssues(l:analyze_output)
+  if 0 == len(l:issues)
+    echo 'Code Climate found no issues! Well done.'
+  else
+    call s:ShowIssues(l:issues)
+  endif
+endfunction
+
+function! s:ParseIssues(text) abort
```

Comment:
I don't think so: I did some research into pure-vimscript JSON parsing when I started thinking about writing this plugin, and I found nothing to encourage me to do it. It seems like the options are either 1) rely on an external tool like Python or 2) use some pure vimscript options but it's very slow and not necessarily a fully spec-compliant parser.

Neither of those options seemed acceptable to me, so I'm going to live with regexes as long as it's reasonable.